File: C++/2.cpp
#include <iostream>

using namespace std;

int main() {
	int prev = 1;
	int current = 2;
	int next = 0;
	int sum = 0;

	while (second < 4000000) {
		next = prev + current;
		prev = current;
		current = next;

		if (prev % 2 == 0) {
			sum += prev;
		}
	}
	cout << sum << endl;
}

File: Python/2.py
fib = [1, 1, 2]
s = 0

while fib[2] < 4*10**6:
	if fib[2] % 2 == 0:
		s += fib[2]
	fib[:] = fib[1:] + [sum(fib[1:])]

print(s)

"""4613732"""